src/fontMetrics.js

```diff
@@ -1,6 +1,3 @@
-/* eslint no-unused-vars:0 */
-
-const Style = require("./Style");
```

Comment:
Nice.  I ended up removing this dep in another PR.  I missed the comment at the top.